mlir/benchmark/programs/MeasurementBasedComputation.cpp

```
@@ -0,0 +1,67 @@
+namespace mqt::benchmark {
+
+using namespace mlir;
+
+namespace {
+/// The number of qubits in the cluster chain.
+constexpr int64_t MBQC_CLUSTER = 4;
+/// The measurement angle that the earlier outcomes flip.
+constexpr double MBQC_ANGLE = std::numbers::pi / 4.0;
+} // namespace
+
+SmallVector<Value> measurementBasedComputation(qc::QCProgramBuilder& b,
```

Cppcheck (2.21.0)

[style] The function 'measurementBasedComputation' is never used.

(unusedFunction)